packages/office-ui-fabric-react/src/components/Fabric/Fabric.test.tsx

```diff
@@ -3,9 +3,21 @@ import * as renderer from 'react-test-renderer';
 import { Fabric } from './Fabric';
 
 describe('Fabric', () => {
-  it('renders correctly', () => {
+  it('renders a Fabric component correctly', () => {
     const component = renderer.create(<Fabric>test</Fabric>);
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('renders a Fabric component with applyTheme correctly', () => {
+    const component = renderer.create(<Fabric applyTheme>test</Fabric>);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders a Fabric component with applyThemeToBody correctly', () => {
```

Comment:
I don't believe this snapshot is useful. Since the snapshot does not encapsulate the body, it's not actually validating any behavior.

Reply:
Spoke offline a bit - seems like it might be hard to test this sort of change to body. Right now, across all the snapshot tests in the Fabric codebase, there aren't any examples of testing the changes to body like the ones happening here but I'm trying to find a way.

Reply:
So the main way to test this sort of change to body seems to be with a hasClass check like this https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Facepile/Facepile.test.tsx#L175, https://airbnb.io/enzyme/docs/api/ReactWrapper/hasClass.html but that's the same as the assert style that was not recommended in a previous comment by @dzearing

Reply:
I researched a lot about using enzyme, even other solutions like puppeteer & tried a bunch of different things like:
>     const dom = mount(<Fabric applyThemeToBody>test</Fabric>);
>     const outermost = dom.childAt(0); 
>     expect(outermost.hasClass('bodyThemed')).toBe(true);
but ended up landing on just getDOMNode() - still not directly rendering into document & using enzyme so that's better than before.